src/PollbotAPI.js

```diff
@@ -7,7 +7,8 @@ import type {
   ReleaseInfo
 } from "./types";
 
-export const SERVER = "https://pollbot.services.mozilla.com/v1";
+export const SERVER =
+  process.env.POLLBOT_URL || "https://pollbot.services.mozilla.com/v1";
```

Comment:
I'm git-confused. This used to be `"https://pollbot.services.mozilla.com/v1"` and now it changed back to `"pollbot.services.mozilla.com/v1"` which is wrong. It needs the `https://` otherwise `fetch` will go to `http://localhost:3000/pollbot.services.mozilla.com/v1/...`

Reply:
why did you see a version without the https?

Reply:
It must have been me looking at a commit and not the whole PR. Or something.